### PR TITLE
Fix for Cisco group device health tab

### PR DESCRIPTION
**Pull request by fbouynot**
Fix for https://github.com/librenms/librenms/issues/14264
DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
It will cause an error on line 53 now

**Comment by fbouynot** [comment]
> It will cause an error on line 53 now

I don't get this error, have you 0 qfp like me or more?

File: includes/html/pages/device/health.inc.php [hunk 1]
```diff
@@ -38,7 +38,9 @@
     $component = new LibreNMS\Component();
     $components = $component->getComponents($device['device_id'], ['type'=> 'cisco-qfp']);
     $components = $components[$device['device_id']];
-    $qfp = count($components);
```
Comment (PipoCanaja) [inline includes/html/pages/device/health.inc.php:41]: Hi
You can replace line 41 with this : 
`$qfp = isset($components) ? count($components) : 0; `

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

File: includes/html/pages/device/health.inc.php (hunk 1 above)
Comment (fbouynot) [reply includes/html/pages/device/health.inc.php:41]: It's changed

**Review by murrant — APPROVED** [review]
Thanks for working on this, you were close, but you needed to leave the $qfp = 0; line

**Review by murrant — APPROVED** [review]

**Pull request merged**